postgres/codenamemap.yaml

```diff
@@ -14,21 +15,26 @@
   {% if repo.use_upstream_repo == true %}
     {% set version = repo.version %}
     {% set fromrepo = repo.fromrepo|default(name ~ '-pgdg', true) %}
+    {% set conf_dir = '/etc/postgresql/' + version|string + '/main' %}
+    {% set data_dir = '/var/lib/postgresql/' + version|string + '/main' %}
   {% else %}
     {% set fromrepo = name %}
+    {% set conf_dir = defaults.conf_dir %}
+    {% set data_dir = defaults.data_dir %}
```

Comment:
I have checked Ubuntu docs on [PostgreSQL](https://help.ubuntu.com/lts/serverguide/postgresql.html), and it appears that "downstream" packages also store all configuration files in `/etc` as well. Tried to install PG on Debian: same thing there.

So we need to separate conf and data directory for the whole Debian OS family.

Reply:
I checked my commits on #187 (which was tested heavily) and you're correct. Fixed.